test/components/views/messages/MPollBody-test.tsx

```diff
@@ -150,6 +151,55 @@ describe("MPollBody", () => {
         expect(voteButton(body, "italian").hasClass(CHECKED)).toBe(false);
     });
 
+    it("cancels my local vote if another comes in", () => {
+        // Given I voted locally
+        const votes = [responseEvent("@me:example.com", "pizza", 100)];
+        const body = newMPollBody(votes);
+        const props: IBodyProps = body.instance().props as IBodyProps;
+        const pollRelations: Relations = props.getRelationsForEvent(
+            "$mypoll", "m.reference", "org.matrix.msc3381.poll.response");
```

Comment:
should be using the constant here. Eg: `POLL_RESPONSE_EVENT_TYPE.name` - this will make it forwards compatible when it eventually switches over to the real value.

applies throughout

Reply:
In tests I prefer to use real strings, so we are totally explicit.  If you disagree, let's chat more about this.

Reply:
The concern with real strings is mostly that it's not clear to a developer what they need to do to fix the tests when trying to switch to stable namespaces - the test failure could be that they broke something, or that they need to update the string. The `Unstable/NamespacedValue` class is tested independently, so should be more than safe to use in other tests.

Reply:
I prefer tests to use as little "magic", code as possible, and be as obvious as they can be about what is really happening.

But, I defer to your experience of pain points, and I fixed this in 3bf34ce88a

Reply:
in the past we've had discussions about how we want to approach testing, but the barrier has always been that testing is difficult. Now that we're a larger team and have actual tests landing (thank you!) we should probably rehash the details for details like this.